commands/documents.ts

```diff
@@ -86,11 +93,20 @@ export const generateEvents = async (n) => {
 };
 
 export const generateGraph = async ({ users = 100, maxHosts = 3 }) => {
-  await alertIndexCheck();
+  //await alertIndexCheck(); TODO
```

Comment:
This function just wasn't defined anywhere

🤷

Reply:
just remove?

Reply:
I'm guessing this means that `generateGraph` is also unused, since it called a undefined function.